workshops/Tkinter Workshop/README.md

```diff
@@ -0,0 +1,221 @@
+
+```python
+import tkinter as tk
+import random
+
+quotelist = [
+
+"Instead of wondering when your \nnext vacation is, maybe you \nshould set up a life you don’t \nneed to escape from.",
+
+"Sometimes\n you win,\n sometimes \nyou learn.",
+
+"Avoiding failure\n is to avoid \nprogress.",
+
+"If the plan \ndoesn’t work, \nchange the plan,\n but never the \ngoal.",
+
+"Creativity \nis \nintelligence \nhaving \nFun."
+
+]
+window = tk.Tk()
+window.minsize(800, 560)
+window.title("Quotes Hub")
+window.config(bg="black")
+tk.Label(window, font=("Helvetica", 60, "bold"),
+ text="QUOTES HUB", bg="black", fg="white").pack()
+tk.Button(window,command=lambda:clicked(random.choice(quotelist)),text="Open",bg="aqua",fg="black",width=20).place(x=315,y=300)
+window.mainloop()
+```
+
+If you run the program by clicking the green "Run" button at the top, the output will be an error, telling us the function `clicked` has not been declared. Let’s fix that by creating that function, above the `window` declaration.
+
+```python
+def clicked(quote):
+F1=tk.Frame(window,bg="light salmon",width=800,height=560)
+F1.place(x=0,y=0)
+```
+
+In this function, we are showing a new page in which a random quote is displayed. We are using the Frame function to create a page here. The Frame function is used to organize a group of widgets. It acts like a container that can be used to hold the other widgets. In that frame, we display the quote at the center using the label function.
+
+```python
+tk.Label(F1,text=quote,font=("Helvetica", 30,"bold"),bg="light salmon",fg="black").pack()
+```
+
+We have to pass the random quote to this function.
+
+```python
+text=quote
+```
+
+Here we gave `text` as the random quote passed from the button.
+
+If you don't understand, just go to the live demo and see the button workflow [here](https://repl.it/@bezlin/graphics#main.py).
+
+Finally the code will look like this:
+
+```python
+import tkinter as tk
+import random
+
+quotelist = [
+
+ "Instead of wondering when your \nnext vacation is, maybe you \nshould set up a life you don’t \nneed to escape from.",
+
+ "Sometimes\n you win,\n sometimes \nyou learn.",
+
+ "Avoiding failure\n is to avoid \nprogress.",
+
+ "If the plan \ndoesn’t work, \nchange the plan,\n but never the \ngoal.",
+
+ "Creativity \nis \nintelligence \nhaving \nFun."
+
+ ]
+
+def clicked(quote):
+ F1=tk.Frame(window,bg="light salmon",width=800,height=560)
+ F1.place(x=0,y=0)
+ tk.Label(F1,text=quote,font=("Helvetica", 30, "bold"),bg="light salmon",fg="black").pack()
+window = tk.Tk()
+window.minsize(800, 560)
+window.title("Quotes Hub")
+window.config(bg="black")
+tk.Label(window, font=("Helvetica", 60, "bold"),
+ text="QUOTES HUB", bg="black", fg="white").pack()
+tk.Button(window,command=lambda:clicked(random.choice(quotelist)),text="Open",bg="aqua",fg="black",width=20).place(x=315,y=300)
+window.mainloop()
+
+```
+
+![run it](https://cloud-5m2nwfs8r.vercel.app/ezgif.com-video-to-gif-4.gif)
+
```

Comment:
Here, can you include a screenshot or gif of you running the program and the final demo apepars?

Reply:
ok